src/cpu/amd/pi/00730F01/update_microcode.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * This file is part of the coreboot project.
+ *
+ * Copyright (C) 2015 Timothy Pearson <tpearson@raptorengineeringinc.com>, Raptor Engineering
+ * Copyright (C) 2007 Advanced Micro Devices, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+#include <stdint.h>
+#include <cpu/amd/microcode.h>
+
+struct id_mapping {
+	uint32_t orig_id;
+	uint16_t new_id;
+};
+
+static u16 get_equivalent_processor_rev_id(u32 orig_id) {
+	static const struct id_mapping id_mapping_table[] = {
+		/* Family 16h */
+
+// TODO This equivalent processor revisions ID needs verification
+                { 0x730f01, 0x7301 },
+
+		/* Array terminator */
+		{ 0xffffff, 0x0000 },
+	};
```

Comment:
@miczyg1 please configure your editor to create correct alignment.

Reply:
@pietrushnic weird I can see the offset here, but when I pull the branch from remote, it looks allright